compiler/src/main/kotlin/visitors/prettyprinter.kt

```diff
@@ -104,7 +104,7 @@ class PrettyPrinter : BaseASTVisitor() {
      */
     override fun visit(node: StateDecl) {
         // Print signature of state. TODO: must take multi-state-declaration into account when implemented
-        stringBuilder.appendln("state ${node.ident} (${node.red}, ${node.green}, ${node.blue}) {")
+        stringBuilder.appendln("state ${node.ident}[${node.multiStateCount}] (${node.red}, ${node.green}, ${node.blue}) {")
```

Comment:
Have we fully agreed that single-state is equivalent to a multi-state with size 1?

Reply:
We haven't talked about it, but functionally they must be equivalent.